react-commenting.js

```diff
@@ -8,18 +8,21 @@ class App extends Component{
   constructor(){
     super()
     // 1.
+    //Defines the default state value of the class object. 
     this.state = {
       userInput: "",
     }
   }
 
   robot = (e) => {
     // 2.
+    //Changes the state mentioned above in the current class. 
```

Comment:
✔️ YES and i LOVE that you made a point to say it's happening in the _current class_